### ISSUE TITLE
Content Search with Sensitive Information Types in Exchange and Teams

### ISSUE THREAD

**Issue by Yarkane** [MicrosoftDocs/microsoft-365-docs#10276]
Hey there.

In this article, it is described that we can:
- Search "for sensitive data types and custom sensitive data types in SharePoint and OneDrive for Business".
- Return "documents on SharePoint and OneDrive for Business sites that contain a" given Sensitive Data Type.

But it does not specify clearly that **you CAN'T** search sensitive data types in Exchange Online or Teams 1:1 messages.

In fact, I remember that this text was in the documentation somewhere:

_You can't use sensitive information types and the SensitiveType search property to search for sensitive data at-rest in Exchange Online mailboxes. This includes 1:1 chat messages, 1:N group chat messages, and team channel conversations in Microsoft teams because all of this content is stored in mailboxes. However, you can use data loss prevention (DLP) policies to protect sensitive email data in transit._

I don't manage to find this text again... Maybe it is not enough present in the documentation. Maybe it has been removed ? If it has been removed, is it possible today to search this in Teams and Exchange ? Then, could this be present in the documentation explicitely ?

So I think it would be nice to place this as a note in multiple places, to be sure the user will catch this important piece of information.

Thanks and have a good day !

---
#### Document Details

⚠ *Do not edit this section. It is required for learn.microsoft.com ➟ GitHub issue linking.*

* ID: e83b77e5-e3ec-6a5f-1f09-e232e2dda483
* Version Independent ID: 0b263c72-95a2-d111-20a7-68857f288369
* Content: [Keyword queries and search conditions for eDiscovery - Microsoft Purview (compliance)](https://learn.microsoft.com/en-us/microsoft-365/compliance/keyword-queries-and-search-conditions?view=o365-worldwide)
* Content Source: [microsoft-365/compliance/keyword-queries-and-search-conditions.md](https://github.com/MicrosoftDocs/microsoft-365-docs/blob/public/microsoft-365/compliance/keyword-queries-and-search-conditions.md)
* Service: **o365-seccomp**
* GitHub Login: @robmazz
* Microsoft Alias: **robmazz**

**Comment by yogkumgit** [comment]
@msbemba Thank you very much for the contribution and sharing this explanation. @Yarkane Hope this comment is helpful for you. If you see a documentation update is required, please feel free to open an issue for the same. We proceed here to close it. Thanks for taking out some time to open the issue. Appreciate and encourage you to do the same in future also.

**Comment by Yarkane** [comment]
Hi, I would like to add a remark there,

I just tried to use Content Search in Exchange, using Sensitive Information Types, and I don't see any change : the results are the same compared to when the warning we discussed was here. 

I don't feel that there are no such limitations anymore. Are there resources about how Content Search in Exchange have evolved, in the MS community for example, explaining why the documentation has changed ? I have not found any.

Thanks a lot for your help.
